packages/components/.storybook/preview.tsx

```diff
@@ -1,16 +1,19 @@
-import "@chanzuckerberg/czedi-kit-styles/normalize.css";
-import "@chanzuckerberg/czedi-kit-styles/all.css";
-
 import * as React from "react";
 
 import { DocsContainer, DocsPage } from "@storybook/addon-docs/blocks";
+import { GlobalStyles } from "twin.macro";
 
 import { addDecorator } from "@storybook/react";
 import { addParameters } from "@storybook/react";
 import { withA11y } from "@storybook/addon-a11y";
 
 addDecorator(withA11y);
-addDecorator((storyFn) => <div dir="ltr">{storyFn()}</div>);
+addDecorator((storyFn) => (
+  <div dir="ltr">
+    <GlobalStyles />
```

Comment:
hm just thinking ahead -- will we need to export components with this attached? or import this in traject?

Reply:
We'll want to re-export, so that traject (and along) can do something like:

```js
// In Traject...

import { GlobalStyleShenanigansThing } from '@chanzuckerberg/eds';
```